translations/base-nl.yaml

```diff
@@ -97,23 +97,23 @@ global:
 
     # The suffix for large numbers, e.g. 1.3k, 400.2M, etc.
     suffix:
-        thousands: k
+        thousands: K
         millions: M
         billions: B
         trillions: T
 
     # Shown for infinitely big numbers
-    infinite: inf
+    infinite: oneindig
```

Comment:
This might be too long. Use a shortening or infinity sign

Reply:
as in the utf8 char for infinity? `∞` (U+221E)

Reply:
Yeah, you can use it if you want